src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.h

```diff
@@ -83,9 +83,10 @@ class DefaultDACVerifier : public DeviceAttestationVerifier
 
     CsaCdKeysTrustStore * GetCertificationDeclarationTrustStore() override { return &mCdKeysTrustStore; }
 
-    void SetRevocationDelegate(DeviceAttestationRevocationDelegate * revocationDelegate)
+    CHIP_ERROR TrySettingRevocationDelegate(DeviceAttestationRevocationDelegate * revocationDelegate) override
```

Comment:
NIT: Why its renamed from `SetRevocationDelegate` to `TrySettingRevocationDelegate`? I do not see any convention in the codebase with Try* naming for setters. The convention is that setter might fail if it returns CHIP_ERROR. "Try" suggests that something might fail temporally, so caller should repeat call attempt after some time. It's not the case here...

Reply:
It was the suggestion from @tehampson in https://github.com/project-chip/connectedhomeip/pull/37134#discussion_r1932304885.

Wasn't aware about the `Try*` convention. Agree, we can keep `SetRevocationDelegate()`.

Reply:
I'm not sure whether we have any formal convention for that but that's how I read it in common APIs like POSIX or glib.